draft-packages/avatar/KaizenDraft/Avatar/AvatarGroup.tsx

```diff
@@ -0,0 +1,84 @@
+import * as React from "react"
+import cx from "classnames"
+import { v4 } from "uuid"
+import { Avatar, GenericAvatarProps, CompanyAvatarProps } from "./Avatar"
+import styles from "./AvatarGroup.module.scss"
+
+export type AvatarProps =
+  | Omit<GenericAvatarProps, "size">
+  | Omit<CompanyAvatarProps, "size">
+
+export type AvatarGroupSize = "small" | "medium" | "large"
+export type AvatarList = [AvatarProps, ...AvatarProps[]]
+
+export interface AvatarGroupProps {
+  /**
+   * There are 3 fixed sizes available. `"small"` will remove border and box shadow to save space.
+   * @default "medium"
+   */
+  size?: AvatarGroupSize
+  /**
+   * If the length of Avatars exceeds the `maxVisible` a counter token will render
+   * @default 2
+   */
+  maxVisible: 2 | 3 | 4 | 5 | 6
+  /**
+   * Takes a array of `AvatarProps` that must have at least item.
+   * Note that 'size' is ommited from the `AvatarProps` type so it will throw a type error if size is provided.
+   * */
+  avatars: AvatarList
+}
+
+const renderCounter = (remainingAvatars: number) => {
+  if (remainingAvatars <= 0) return
+  return (
+    <li
+      aria-label={`There is ${remainingAvatars} other memeber${
+        remainingAvatars > 1 && "s"
+      } of this group`}
+    >
+      <span
+        className={styles.AvatarCounter}
+        aria-hidden={true}
+        role="presentation"
+      >
+        {`+${remainingAvatars}`}
+      </span>
+    </li>
+  )
+}
+
+const renderAvatars = (
+  avatars: AvatarList,
+  maxVisible: number,
+  size: AvatarGroupSize
+) => (
+  <>
+    {avatars?.map(
+      (avatarProps, index) =>
+        index < maxVisible && (
+          <li
+            key={`avatar-${v4()}`}
+            className={styles.AvatarGroupItem}
+            role="presentation"
```

Comment:
I think this role="presentation" was left over from when I was experimenting with what a screen readers would read aloud when iterating over this loop to see if I could get a fluid read of all members. It shouldn't really be here since it detract from what a list should be read as so I'm going to remove it.